README.md

```diff
@@ -1,15 +1,15 @@
+# ![recheck logo](https://user-images.githubusercontent.com/1871610/41766965-b69d46a2-7608-11e8-97b4-c6b0f047d455.png)
+
 [![Build Status](https://travis-ci.com/retest/recheck.svg?branch=master)](https://travis-ci.com/retest/recheck)
 [![license](https://img.shields.io/badge/license-AGPL-brightgreen.svg)](https://github.com/retest/recheck/blob/master/LICENSE)
 [![PRs welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg)](https://github.com/retest/recheck/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 [![code with hearth by retest](https://img.shields.io/badge/%3C%2F%3E%20with%20%E2%99%A5%20by-retest-C1D82F.svg)](https://github.com/retest)
 
-# recheck
-
 recheck is a general framework that allows replacing manual asserts and checking everything at once.
 
 ## Features
 
-* Easy creation and maintenance of checks for web, XML and Java Swing (more coming).
+* Easy creation and maintenance of checks for [web](https://github.com/retest/recheck-web/), XML and [Java Swing](https://retest.de/en/) (more coming).
```

Comment:
Hmmm, list XML although it's not publicly available yet? Also, the last bullet point ("The Git for your interface") then should be adapted as well since XML isn't really an interface.